### PR TITLE
Feature/log handling

### PR DESCRIPTION
**Pull request by sinaure**
improved log handling using graylog compatible formatting

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -339,14 +337,10 @@ def OSMInstantiationGetIPs(service_platform,id):
 
 @app.route('/adapters/<service_platform>/instantiations', methods=['POST'])
 def serviceInstantiation(service_platform):
-    LOG.debug(request.is_json)
-    LOG.debug(request)
+    LOG.info("dsjfjsdhgfjsdhfg")
```
Comment (felipevicens) [inline main.py]: Please remove this log

File: docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,18 @@
+version: "3"
+services:
+  pg:
+    image: postgres:10.0-alpine
```
Comment (felipevicens) [inline docker-compose.yml]: We use sonatanfv/son-postgres as postgres image

File: docker-compose.yml [hunk 3]
```diff
@@ -0,0 +1,18 @@
+version: "3"
+services:
+  pg:
+    image: postgres:10.0-alpine
+    ports:
+      - 5432:5432
+    volumes:
+      - pg_data:/var/lib/postgresql/data/pg_data
+      - ./init.sql:/docker-entrypoint-initdb.d/init.sql
```
Comment (felipevicens) [inline docker-compose.yml]: It is initialized when the tng-vnv-platform-adapter starts 
Check here:
https://github.com/sonata-nfv/tng-vnv-platform-adapter/blob/master/main.py#L602

File: .gitignore [hunk 4]
```diff
@@ -102,3 +102,7 @@ venv.bak/
 
 # mypy
 .mypy_cache/
+
```
Comment (felipevicens) [inline .gitignore:105]: can you add .vscode?

**Review by felipevicens — COMMENTED** [review]
Some reviews before merge

File: main.py (hunk 1 above)
Comment (sinaure) [reply main.py]: ok

File: docker-compose.yml (hunk 2 above)
Comment (sinaure) [reply docker-compose.yml]: ok

File: docker-compose.yml (hunk 3 above)
Comment (sinaure) [reply docker-compose.yml]: schema is initialized but there are no data

File: .gitignore (hunk 4 above)
Comment (sinaure) [reply .gitignore:105]: ok

**Pull request merged**